operator/validator/controller.go

```diff
@@ -436,43 +436,59 @@ func (c *controller) handleNonCommitteeMessages(
 	return nil
 }
 
-// StartValidators loads all persisted shares and setup the corresponding validators
-func (c *controller) StartValidators(ctx context.Context) {
+// StartValidators loads all persisted shares and sets up the corresponding validators
+func (c *controller) StartValidators(ctx context.Context) error {
 	// TODO: Pass context whereever the execution flow may be blocked.
 
-	// Load non-liquidated shares.
-	shares := c.sharesStorage.List(nil, registrystorage.ByNotLiquidated())
-	if len(shares) == 0 {
+	if c.validatorCommonOpts.ExporterOptions.Enabled {
+		// There are no committee validators to set up.
 		close(c.committeeValidatorSetup)
-		c.logger.Info("could not find validators")
-		return
+		return nil
 	}
 
-	var ownShares []*ssvtypes.SSVShare
-	for _, share := range shares {
-		if c.operatorDataStore.GetOperatorID() != 0 && share.BelongsToOperator(c.operatorDataStore.GetOperatorID()) {
-			ownShares = append(ownShares, share)
+	init := func() ([]*validator.Validator, []*validator.Committee, error) {
+		defer close(c.committeeValidatorSetup)
+
+		// Load non-liquidated shares that belong to our own Operator.
+		ownShares := c.sharesStorage.List(
+			nil,
+			registrystorage.ByNotLiquidated(),
+			registrystorage.ByOperatorID(c.operatorDataStore.GetOperatorID()),
+		)
+		if len(ownShares) == 0 {
+			c.logger.Info("no own non-liquidated validator shares found in DB")
```

Comment:
If we are writing a info log, I think it's appropriate to at least add something simpler and more "user-oriented" like - "no validators to start" and then specify the reason like you did.